testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -6857,7 +6857,6 @@ fn check_block_times() {
     let info = get_chain_info_result(&naka_conf).unwrap();
     info!("Chain info: {:?}", info);
     let last_stacks_block_height = info.stacks_tip_height as u128;
-    let last_tenure_height = last_stacks_block_height as u128;
```

Comment:
Since `get-tenure-info?` takes a stacks block height as the second argument, I find it confusing to have a second variable named `last_tenure_height` here